src/test/groovy/org/osiam/resources/provisioning/SCIMUserProvisioningBeanSpec.groovy

```diff
@@ -31,15 +31,16 @@ import org.osiam.storage.dao.UserDao
 import org.osiam.storage.entities.MetaEntity
 import org.osiam.storage.entities.UserEntity
 import org.springframework.security.authentication.encoding.PasswordEncoder
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder
 import spock.lang.Specification
 
 class SCIMUserProvisioningBeanSpec extends Specification {
 
-    def passwordEncoder = Mock(PasswordEncoder)
+    BCryptPasswordEncoder bCryptPasswordEncoder = Mock()
```

Comment:
Why did you change it to this? The replaced statement is the spock idiom to mocking.